### DISCUSSION TITLE
Add concept of failing beatmaps to multiplayer

### DISCUSSION THREAD

**Discussion by timiimit** [ppy/osu#23594] [Ideas]
Stable has a concept of failing in multi. Since lazer is supposed to have everything from stable and more, I think this feature should be implemented.

I think a good way of doing this would be to add a fail text to skins. Text that would show only when you have failed.

It is also worth thinking about making a persistent note on the screen that player has failed. So instead of just showing fail text for a short time, make it permanent until the end of the beatmap.

**Comment by bdach** [comment]
Probably should just use F rank to denote failed plays honestly.

**Reply by timiimit** [reply]
Personally I just really like some feedback exactly at the time of fail. But that's an alright solution i suppose.
